VERSION

```diff
@@ -1,4 +1,4 @@
-VERSION="2.11.6"
+VERSION="2.11.7"
```

Comment:
* on 2.12 branch, `VERSION` needs to be `2.12.*`
* this is not a 2.12.0, but only a preview (if anything)

Reply:
I created the preview 1 of the 2.12.0. Preview 2 will (hopefully) be with multi shard support.